doc/user_guide/examples.rst

```diff
@@ -249,6 +249,24 @@ within an OpenACC Parallel directive and 4) there are no checks on
 whether loops are parallel or not, it is just assumed they are -
 i.e. support for colouring or locking is not yet implemented.
 
+Example 15: CPU Optimisation of Matvec
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Example of optimising the LFRic matvec kernel for CPUs. This is work
+in progress with the idea being that PSyclone transformations will be
+able to reproduce hand-optimised code.
+
+There is one script which, when run
```

Comment:
This isn't formatted correctly: should be "run::"

Reply:
oops